README.md

```diff
@@ -59,21 +59,24 @@ buildscript {
 You can now run commands like `./gradlew appengineDeploy` in the root folder of your Java application.
 
 # Supported tasks
-- appengineStage
-- appengineDeploy
-- appengineDeployCron
-- appengineDeployDispatch
-- appengineDeployDos
-- appengineDeployIndex
-- appengineDeployQueue
+| Goal                       | Description
```

Comment:
nit: are these called goals or tasks (I thought the gradle terminology was tasks)? @loosebazooka

Reply:
Yeah, I think they're tasks. Was referencing the maven plugin readme and forgot to change this.